src/pybind/mgr/cephadm/serve.py

```diff
@@ -1539,13 +1545,12 @@ async def _get_container_image_info(self, image_name: str) -> ContainerInspectIn
             await self._registry_login(host, json.loads(str(self.mgr.get_store('registry_credentials'))))
 
         j = None
-        if not self.mgr.use_repo_digest:
-            try:
-                j = await self._run_cephadm_json(host, '', 'inspect-image', [],
-                                                 image=image_name, no_fsid=True,
-                                                 error_ok=True)
-            except OrchestratorError:
-                pass
+        try:
+            j = await self._run_cephadm_json(host, '', 'inspect-image', [],
+                                             image=image_name, no_fsid=True,
+                                             error_ok=True)
+        except OrchestratorError:
+            pass
```

Comment:
this part seems to be undoing a change from a recent PR. Was this on purpose? If so, what's the reasoning?

Reply:
Apologize for my mistake. Redo this already.